src/app/globals.css

```diff
@@ -31,3 +31,22 @@ body {
     text-wrap: balance;
   }
 }
+
+::-webkit-scrollbar {
+  width: 5px;
+}
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/CSS/::-webkit-scrollbar

This doesn't work on all browsers.
https://caniuse.com/?search=-webkit-scrollbar

Reply:
applied additional css for other browsers